awx/api/views/__init__.py

```diff
@@ -364,6 +364,7 @@ class InstanceList(ListAPIView):
     model = models.Instance
     serializer_class = serializers.InstanceSerializer
     search_fields = ('hostname',)
+    ordering = ('id',)
```

Comment:
The default ordering of the model is by hostname

https://github.com/ansible/awx/blob/56f51eebce7004630394c1c76a15eaebd652072e/awx/main/models/ha.py#L142

I don't see any reason to introduce this ordering by id.

Reply:
doesn't this need to be `id` so that the sorting works with other drop-downs/views?